Tests/SafeDICoreTests/TypeDescriptionTests.swift

```diff
@@ -120,17 +120,28 @@ final class TypeDescriptionTests: XCTestCase {
 
     func test_typeDescription_whenCalledOnATypeSyntaxNodeRepresentingAOptionalTypeSyntax_findsTheType() throws {
         let content = """
-            protocol FooBar: Foo where Something == AnyObject? {}
+            var optionalAnyObject: AnyObject?
```

Comment:
simplifying this test case allowed for a much simpler `OptionalTypeSyntaxVisitor` implementation